emsdk.py

```diff
@@ -1463,6 +1463,19 @@ def load_em_config():
     except:
       pass
 
+def find_emscripten_root(active_tools):
+  """Find the currnetly active emscripten root.
+
+  If there is more than one tool that defines EMSCRIPTEN_ROOT (this
+  should not happen under normal circumstances), assume the last one takes
+  precidence.
+  """
+  for tool in active_tools:
+    config = tool.activated_config()
```

Comment:
```suggestion
    root = None
    config = tool.activated_config()
```
I think without that Python will error if none is found (the `return root` will say it is read before written).